tests/bluetooth/audio/mocks/include/iso.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MOCKS_ISO_H_
+#define MOCKS_ISO_H_
+
+#include <zephyr/fff.h>
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/iso.h>
+#include <zephyr/net/buf.h>
```

Comment:
Ditto with just `iso.h` here? 

How come this includes `fff.h`, but the GATT files do not?

Reply:
fff.h in gatt.h was missing